### DISCUSSION TITLE
Gateway discussion

### DISCUSSION THREAD

**Discussion by codetheweb** [codetheweb/tuyapi#408] [Ideas]
Started from https://github.com/codetheweb/tuyapi/issues/389.

Mentioning @rospogrigio, @Apollon77, @msillano, and @presidentio for notifications. Feel free to unsubscribe if you don't want to follow this thread.

**Comment by BarryW55** [comment]
@codetheweb - you mentioned in another post on the 17th Jan you did not have time right now to address this issue, do you have idea if/when you might have time ?
Many thanks

**Reply by codetheweb** [reply]
Are you talking about gateway support or 0d support?

**Reply by BarryW55** [reply]
@codetheweb I am talking about the 0d support

Thanks

**Reply by codetheweb** [reply]
This discussion is for something else.

https://github.com/codetheweb/tuyapi/issues/389#issuecomment-793144593
